lib/features/modeling/behavior/SubProcessPlaneBehavior.js

```diff
@@ -240,6 +240,27 @@ export default function SubProcessPlaneBehavior(
   }, true);
 
 
+  // ensure the collapsed shapes name is rerendered when the name is changed
+  // through the plane object
+  eventBus.on('element.changed', function(e) {
+    var plane = e.element;
+
+    if (!isPlane(plane)) {
+      return;
+    }
+
+    var collapsedShape = elementRegistry.get(getShapeIdFromPlane(plane));
+
+    // don't re-throw `element.changed` events for the process object,
+    // as no collapsed shape exists to update
+    if (!collapsedShape || collapsedShape === plane) {
+      return;
+    }
+
+    eventBus.fire('element.changed', { element: collapsedShape });
```

Comment:
No, we actually have to fire this one to trigger a re-render in `diagram-js`. Otherwise, the collapsed shape will still display the old name until moved

Reply:
Alright, thanks for clarifying. 💹